pkg/ottl/ottlfuncs/func_parse_json_test.go

```diff
@@ -82,20 +83,32 @@ func Test_ParseJSON(t *testing.T) {
 					return `{"test":["string","value"]}`, nil
 				},
 			},
-			want: func(expectedMap pcommon.Map) {
+			wantMap: func(expectedMap pcommon.Map) {
 				emptySlice := expectedMap.PutEmptySlice("test")
 				emptySlice.AppendEmpty().SetStr("string")
 				emptySlice.AppendEmpty().SetStr("value")
 			},
 		},
+		{
+			name: "handle top level array",
+			target: ottl.StandardStringGetter[any]{
+				Getter: func(_ context.Context, _ any) (any, error) {
+					return `["string","value"]`, nil
+				},
+			},
+			wantSlice: func(expectedSlice pcommon.Slice) {
+				expectedSlice.AppendEmpty().SetStr("string")
+				expectedSlice.AppendEmpty().SetStr("value")
+			},
+		},
```

Comment:
I think that it would be nice to have a test that parses an array of JSON objects as expected in the original issue

Reply:
alright, I have added such a test

Reply:
Thanks @bacherfl. It works as expected!